Server/src/Server.Infrastructure/Persistence/Extensions.cs

```diff
@@ -39,7 +39,7 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, I
             options.UseSqlServer(sqlServerSettings.ConnectionString, o => o.UseQuerySplittingBehavior(QuerySplittingBehavior.SplitQuery));
             if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
             {
-                options.LogTo(Console.WriteLine, LogLevel.Information);
+                // options.LogTo(Console.WriteLine, LogLevel.Information);
```

Comment:
to można wywalić albo odkomentować w takim razie

Reply:
O zapomniałem o tym. Ogólnie był jakiś powód czemu to logowanie jest dodane, zamiast zmian poziomu logowania dla ef core w appsettingsach? @DontEatRice

Reply:
bo zapomniałem ze jak masz DbContext dodany przez AddDbContext do serwisów to się automtycznie zapina logowanie Xd